### PR TITLE
update .htacces and readme.md

### PR DESCRIPTION
**Pull request by Iniely**

### CODE REVIEW

**Comment by dgarijo** [comment]
@Iniely,
please add a readme file stating who will be the maintainer of the repository and a small description, as indicated in the [guidelines](https://github.com/perma-id/w3id.org#suitable-pr-content)

File: aipo/.htaccess [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+
+
+
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^aipo$ http://purl.org/aipo [R=303,L]
```
Comment (davidlehn) [inline aipo/.htaccess:26]: Rules are relative to the current directory so this will result in https://w3id.org/aipo/aipo. You might want `^$`.

**Comment by dgarijo** [comment]
@Iniely,
you are writing a bunch of conditions (lines 15-19) with no redirection afterwards. Aren't you missing the right link (to the html)?
The second redirect is a google drive redirect, which does not return any vocabulary, as you  may seem to want

**Comment by davidlehn** [comment]
@Iniely Also please at the maintainer github id(s) to the readme.

File: aipo/.htaccess [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+AddType application/ld+json .json
+# Rewrite engine setup
+RewriteEngine On
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://lsarni.github.io/AipO/ [R=303,L]
```
Comment (davidlehn) [inline aipo/.htaccess:19]: Should this be https://iniely.github.io/AipO/?

File: aipo/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# AipO
+Administration and International Business Pensum's Ontology from Peruvian Union University 
+
+More info at https://w3id.org/aipo/aipo
```
Comment (davidlehn) [inline aipo/README.md:4]: I don't think that will match any rules, did you mean to just use https://w3id.org/aipo?

File: aipo/.htaccess (hunk 2 above)
Comment (Iniely) [reply aipo/.htaccess:19]: Yes, it should be. I'm going to change it. :)

**Review by davidlehn — APPROVED** [review]

**Comment by davidlehn** [comment]
@Iniely I merged a bit quick.  Could you also please make a new PR adding your github id as a maintainer to the README.  Thanks.

**Pull request merged**